mobile-app/src/views/Settings.js

```diff
@@ -20,12 +20,17 @@ import { StatusBar } from "expo-status-bar";
 import React from "react";
 import { StyleSheet, Text, View } from "react-native";
 import { createStackNavigator } from "@react-navigation/stack";
-import screenHeader from "../components/Header";
+import Header from "../components/Header";
 const SettingsStack = createStackNavigator();
 
 function SettingsStackScreen() {
     return (
-        <SettingsStack.Navigator screenOptions={screenHeader}>
+        <SettingsStack.Navigator
+            screenOptions={{
+                headerTitle: () => <Header />,
+                headerStyle: styles.headerStyle,
+            }}
+        >
             <SettingsStack.Screen name="Home" component={Settings} />
```

Comment:
Settings not Home

Reply:
I'm resolving this conversation and re-requesting a review.